bindings/binding_typescript_wasm/__tests__/transform.js

```diff
@@ -138,5 +138,24 @@ describe("transform", () => {
                 })
             ).rejects.toMatchSnapshot();
         });
+
```

Comment:
Unrelated to this PR, I'm surprised there is no test for Parameter Properties or "import/export equals" or angle-brackets-type-assertion in this file, given the enum and namespace cases are covered here.

Reply:
Actually main tests are done at https://github.com/swc-project/swc/tree/caa7f370ff2003983a3448a2b7e6f0f6d6224b86/crates/swc_fast_ts_strip/tests/fixture

This file tests only the JS interface